ui/app/styles/components/console-ui-panel.scss

```diff
@@ -11,7 +11,7 @@
   min-height: 0;
   overflow: scroll;
   right: 0;
```

Comment:
I tried to replace the hardcoded bulma spacing with our own spacing vars (using equivalent px) but it came back to haunt me. Reverted back to hardcoded bulma spacing with rem values.